docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -283,3 +283,54 @@ GET /transactions/{tx_id}
 
    :statuscode 200: A transaction with that ID was found.
    :statuscode 404: A transaction with that ID was not found.
+
+
+GET /unspents/
```

Comment:
**tl;dr** We'll stick to `/unspents`

A small summary of an offline conversation between @TimDaub and @sbellem:

As @TimDaub pointed out the endpoint does not strictly return `conditions`, but `TransactionLinks`, which from the point of view of relational data modelling, could be seen as a kind of "intermediate table". That is, one could think of conditions, fulfilments and transactions living in separate tables, along with intermediate tables to keep track of their relationships.

In such scenarios, querying can be done in various ways: on transactions, on conditions, or on the intermediate table. 

In this case, `unspents` is the intermediate path. That is we could be querying on either `/transactions`, `/conditions` or the intermediate table resource, which in this case is most commonly known as `unspents`.

So, we'll stick to `/unspents`.